lib/AvatarPresence/index.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import OverlayTrigger from 'react-bootstrap/lib/OverlayTrigger';
+import Tooltip from '../Tooltip';
+
+const tooltipOnHover = (data, isAssigned) => (
+  <Tooltip className="avatar__tooltip" id={data.email}>
+    <p className="avatar__tooltip-name">{data.name}</p>
+    <p className="avatar__tooltip-email">{data.email}</p>
+    { isAssigned &&
+      <span className="avatar__pillbox">Assigned</span>
+    }
+  </Tooltip>
+);
+
+const AvatarPresence = (props) => {
```

Comment:
That's exactly what I wanted to discuss :) and why I left it this way on purpose, in fact. 
The way I see it we can remove the simple Avatar component that's still currently there (just a wrapper for an `img` tag), and replace it with this one (would still be called `Avatar`).

I like the `AvatarWithTooltip` approach, I'll make the move in that direction 👍